client/src/components/Overview.jsx

```diff
@@ -0,0 +1,215 @@
+      console.log(sizesArr)
+      this.setState({sizes: sizesArr})
+
+
+      // initialize quantity of default style and selected size
+      var style = res.data.results[0]
+      var styleSkus = style.skus
+      console.log('styleSkus', styleSkus)
+      var quantity = 0;
+      for (var sku in styleSkus) {
+        console.log('state',this.state.selectedSize)
+        console.log(styleSkus[sku].size)
+        if (styleSkus[sku].size === this.state.selectedSize) {
+          quantity += styleSkus[sku].quantity
+        }
+      }
+      this.setState({quantity:quantity})
+
+
+      // render thuumbnails and available sizes of default style
+
+      var urls = [];
+      console.log(res.data.results[0])
+
+      res.data.results.forEach( (style, i) => {
+        urls.push(   {url: style.photos[0]['thumbnail_url'], index: i})
+      })
+      this.setState({thumbnails: urls})
+    })
+
+
+  }
+
+
+  handleStyleClick(e) {
+
+    // update available sizes upon clicking new style
+    this.setState({selectedStyle : event.target.getAttribute("id")}, () => {
+    var style = this.state.results[Number(this.state.selectedStyle)];
+    var styleSkus = style.skus;
+    var sizesArr = [];
+    for (var sku in styleSkus) {
+      if (!sizesArr.includes(styleSkus[sku].size)) {
+        sizesArr.push(styleSkus[sku].size)
+      }
+    }
+    console.log(sizesArr)
+    this.setState({sizes: sizesArr})
+
+
+    // toggle "outOfStock" boolean if quantity of currently selected style === 0
+
+    var quantity = 0;
+      for (var sku in styleSkus) {
+        quantity += styleSkus[sku].quantity
+      }
+      if (quantity === 0) {
+        this.setState({outOfStock : true})
+      } else {
+        this.setState({outOfStock : null})
+      }
+    })
+  }
+
+
+  handleSizeSelect(e) {
+    console.log(e.target.value)
+
+    // update selectedSize state back to null if user presses "Select Size"
+    if (e.target.value === 'Select Size') {
+      this.setState({selectedSize: null})
+      return;
+    }
+
+    // update selectedSize state and render quantity of given style/size combination selected
+    this.setState({selectedSize: e.target.value}, () => {
+      var style = this.state.results[this.state.selectedStyle]
+      var styleSkus = style.skus
+      console.log('styleSkus', styleSkus)
+      var quantity = 0;
+      for (var sku in styleSkus) {
+        console.log('state',this.state.selectedSize)
+        console.log(styleSkus[sku].size)
+        if (styleSkus[sku].size === this.state.selectedSize) {
+          quantity += styleSkus[sku].quantity
+        }
+      }
+      this.setState({quantity:quantity})
+      })
+
+
+
+    }
+
+
+    handleAddToCart() {
+      // check for invalid add to cart (ATC)
+      if (this.state.selectedSize === null || this.state.selectedStyle === null) {
+        this.setState({validATC: false})
+      } else {
+        this.setState({validATC: true})
+      }
+    }
+
+
+
+
+  render () {
+
+
+    return (
+      <>
```

Comment:
make sure to use divs instead of anonymous tags so that you can add classNames to them for styling purposes.